packages/marko-web/components/page/metadata/content.marko

```diff
@@ -87,6 +90,9 @@ ${processedFragment}
 `;
 
 $ const buildStructuredData = isFunction(input.buildStructuredData) ? input.buildStructuredData : defaultBuildStructuredData;
+$ const defaultFn = ({ content }) => get(content, 'userRegistration.isCurrentlyRequired', false);
```

Comment:
This should not destructure `content` from the first argument, or L50 needs to change to send `{ content: node }` instead of `node` as the first argument. Not sure which one is correct -- where is the contentGatingHandler definition?